core/variant_op.cpp

```diff
@@ -341,6 +341,61 @@ bool Variant::booleanize() const {
 		_RETURN_FAIL                                                                                                                               \
 	}
 
+#define DEFAULT_OP_LOCALMEM_NUM_DIV(m_prefix, m_op_name, m_name, m_op, m_type)                                                                                                                                                                                      \
+	CASE_TYPE(m_prefix, m_op_name, m_name) {                                                                                                                                                                                                                        \
+		if (p_b.type == INT) {                                                                                                                                                                                                                                      \
+			if (p_b._data._int == 0) {                                                                                                                                                                                                                              \
+				r_valid = false;                                                                                                                                                                                                                                    \
+				_RETURN("Division By Zero");                                                                                                                                                                                                                        \
+			}                                                                                                                                                                                                                                                       \
+			_RETURN(*reinterpret_cast<const m_type *>(p_a._data._mem) m_op p_b._data._int);                                                                                                                                                                         \
+		}                                                                                                                                                                                                                                                           \
+		if (p_b.type == FLOAT) {                                                                                                                                                                                                                                    \
+			if (p_b._data._float == 0) {                                                                                                                                                                                                                            \
```

Comment:
If you want to do these tests, at least use the unlikely() function to wrap these conditions.

Reply:
Maybe the checks should all be enclosed in `#ifdef DEBUG_ENABLED` too to avoid the potential performance hit on release builds.

Reply:
I added `unlikely()` and also `#ifdef DEBUG_ENABLED`